docker/Dockerfile.gpu

```diff
@@ -16,10 +16,17 @@ RUN apt-get update && \
     && curl -O https://bootstrap.pypa.io/get-pip.py \
     && python get-pip.py
 
-RUN pip install mxnet-model-server
+RUN pip install mxnet-model-server \
+    && pip install mxnet-cu90mkl
+
+COPY dockerd-entrypoint.sh /usr/local/bin/dockerd-entrypoint.sh
+RUN chmod +x /usr/local/bin/dockerd-entrypoint.sh
```

Comment:
Why not just run "mxnet-model-server" from the ENTRYPOINT?

Reply:
1. use dockerd-entrypoint.sh is by convention
2. if use mxnet-model-server as ENTRYPOINT, it would be hard for user to override it, for example: "docker run -itd mms /bin/bash" won't work in that case.
3. ENTRYPOINT script should used for general purpose, and CMD is the parameter passed to the script.